tests/cases/fourslash/moveToNewFile_inferQuoteStyle.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: /a.ts
+////import 'unrelated';
```

Comment:
Naively, I would have expected to see a test for each possible inference (i.e. one single, one double).

Reply:
Most other `moveToNewFile` tests are using double quotes, since that's the default inference.